src/Api/ProductMediaFileApi.php

```diff
@@ -130,14 +130,14 @@ public function download(string $code): ResponseInterface
      */
     protected function extractCodeFromCreationResponse(ResponseInterface $response)
     {
-        $headers = $response->getHeaders();
+        $locationHeaders = $response->getHeader('Location');
 
-        if (!isset($headers['Location'][0])) {
+        if (!isset($locationHeaders[0])) {
```

Comment:
we could also use the `hasHeader()` method

Reply:
I've updated the code to use the `hasHeader()` helper. Thanks!